flow/layers/layer_tree.cc

```diff
@@ -78,11 +78,14 @@ void LayerTree::Paint(CompositorContext::ScopedFrame& frame,
     }
   }
 
+  MutatorsStack* stack = new MutatorsStack();
```

Comment:
Should we only add this if user enabled platform views? If so, what is the flag that i need to check?

Reply:
done